src/core/core.controller.js

```diff
@@ -147,8 +147,9 @@ function updateConfig(chart) {
 	chart.tooltip.initialize();
 }
 
-function positionIsHorizontal(position) {
-	return position === 'top' || position === 'bottom';
+const WELL_KNOWN_POSITIONS = ['top', 'bottom', 'left', 'right', 'chartArea'];
```

Comment:
I might call this just `KNOWN_POSITIONS`

Reply:
Done